atomkraft/test/__init__.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Optional
+import typer
+
+app = typer.Typer(rich_markup_mode="rich", add_completion=False)
+
+def FileOption(help, default):
+    return typer.Option(None, show_default = False, 
+        help = f"{help} [grey30]\[default: set via [bold cyan]atomkraft {default}[/bold cyan]][/grey30]")
+
+@app.command()
+
```

Comment:
This extra empty line can be removed.

Reply:
Thanks, Rano! Done; it reformatted the code quite substantially:)